core/runtime/src/main/java/io/quarkus/runtime/ConfigConfig.java

```diff
@@ -0,0 +1,30 @@
+package io.quarkus.runtime;
+
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+/**
+ * We don't really use this, because these are configurations for the config itself, so it causes a chicken / egg
+ * problem, but we have it so the configurations can be properly documented.
+ *
+ * Relocation of the Config configurations to the Quarkus namespace is done in ConfigUtils#configBuilder.
+ */
+@ConfigRoot(name = ConfigItem.PARENT, phase = ConfigPhase.BUILD_AND_RUN_TIME_FIXED)
+public class ConfigConfig {
```

Comment:
Didn't see that one coming :)

Reply:
LOL!

Reply:
You got a better name? :P

Reply:
Nope :). I let you mark it ready for review once you think it's ready!

Reply:
Sure :)